packages/SwingSet/src/kernel/kernelSyscall.js

```diff
@@ -28,13 +28,14 @@ export function doSend(kernelKeeper, target, msg) {
 export function makeKernelSyscallHandler(tools) {
   const {
     kernelKeeper,
-    kvStore,
     ephemeral,
     notify,
     doResolve,
     setTerminationTrigger,
   } = tools;
 
+  const { kvStore } = kernelKeeper;
```

Comment:
Having `kernelKeeper` hold the `kvStore` raises the question in my mind if this refactoring means it would be better to no longer have `kernelSyscall.js` do `kvStore` operations directly at all, or if everything that touches the `kvStore` should go through `kernelKeeper`.  I don't think I'm ready to argue wholeheartedly for that yet, but it seems like we might be moving in that direction.

Reply:
Yeah, I think you're right: `kernelKeeper` imposes the schema, so everything should go through it. The only thing `kernelSyscall.js` is using `kvStore` for is the `vatstore(Get/Set/Delete)` syscalls, and all three (plus `vatstoreKeyKey`) should move into `kernelKeeper` so that the `${vatID}.vs.${key}` key-construction logic can live next to the rest of the key-construction logic.